woohoo.rb

```diff
@@ -0,0 +1,10 @@
+(1..10).reduce(&:+)
+(1..10).inject(&:+)
+
+'super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line'
+"super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line"
```

Comment:
Literal `"super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line"` used in void context.<br>Line is too long. [182/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.